tests/test_int2bool_pysat.py

```diff
@@ -0,0 +1,190 @@
+import unittest
+import cpmpy as cp 
+from cpmpy.expressions import *
+from cpmpy.model import Model
+from cpmpy.solvers.pysat import CPM_pysat
+from cpmpy.transformations.to_cnf import to_cnf
+from cpmpy.transformations.int2bool_onehot import to_bool_constraint
+import numpy as np
+
+class TestInt2BoolPySAT(unittest.TestCase):
+    def test_base_bool_model(self):
+        iv = intvar(lb=3, ub=7)
+
+        m = Model(
+            iv > 4
+        )
+        s = CPM_pysat(m)
+        s.solve()
+    
+    def test_incremental_int2bool_model(self):
+        iv1 = intvar(lb=3, ub=7)
+        iv2 = intvar(lb=3, ub=5)
+
+        m = Model(
+            iv1 > 6
+        )
+
+        s = CPM_pysat(m)
+        s.solve()
+
+        s += iv2 < 4
+        s.solve()
+    
+    # abs() not implemented yet
```

Comment:
its very good to add test cases too; lots of things here in comments though, remove or activate?